### PR TITLE
docs(readme): add missing link to home-manager

### PR DESCRIPTION
**Pull request by kidonng**
#### Description

The link to `home-manager` seems to be missing when the section was added.

#### Motivation and Context

Closes #

#### How Has This Been Tested?

- [ ] I have tested using **MacOS**
- [ ] I have tested using **Linux**
- [ ] I have tested using **Windows**

#### Checklist:

- [ ] I have updated the documentation accordingly.
- [ ] I have updated the tests accordingly.

### CODE REVIEW

File: docs/installing/README.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ curl -fsSL https://starship.rs/install.sh | bash -s -- -b /data/data/com.termux/
 nix-env -iA nixos.starship
 ```
 
-#### Declarative, single user, via [home-manager](home-manager)
+#### Declarative, single user, via [home-manager](https://github.com/nix-community/home-manager)
```
Comment (vladimyr) [inline docs/installing/README.md:52]: Linking to the Github repo is fine but other links point either to frontpages or wikis. In this case, we could follow the suit by linking to the manual https://nix-community.github.io/home-manager
@kidonng What do you think?

Comment (kidonng) [reply docs/installing/README.md:52]: I'm aware of the manual, but it directly jumps to installation and usage without explaining what is it and "Words of warning" like the README on GitHub does. This is friendlier for those who may not be familiar with what the `home-manager` is.

Comment (vladimyr) [reply docs/installing/README.md:52]: > This is friendlier for those who are not familiar with `home-manager.

That is exactly what I wanted to make sure, thanks!

**Review by vladimyr — APPROVED** [review]

**Comment by chipbuster** [comment]
@kidonng Thanks for the fix!

**Pull request merged**